pkgs/by-name/ba/basicswap/package.nix

```diff
@@ -0,0 +1,104 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  linkFarm,
+  secp256k1,
+  particl-core,
+  bitcoind,
+  namecoind,
+  monero-cli,
+  wownero,
+}:
+
+let
+  secp256k1_anonswap = secp256k1.overrideAttrs (old: {
+    src = fetchFromGitHub {
+      owner = "tecnovert";
+      repo = "secp256k1";
+      rev = "fd8b63ccf8bcb48358a42c456f34e2488a55a688";
+      hash = "sha256-/bmKZRBBjirI4YqRKfzoxdAt6UVoWHmrNQQHX7l+eH8=";
+    };
+    configureFlags = old.configureFlags ++ [
+      "--enable-experimental"
+      "--enable-module-ed25519"
+      "--enable-module-generator"
+      "--enable-module-dleag"
+      "--enable-module-ecdsaotves"
+    ];
+  });
+  coincurve-anonswap =
+    (python3Packages.coincurve.override {
+      secp256k1 = secp256k1_anonswap;
+    }).overrideAttrs
+      (old: {
+        src = fetchFromGitHub {
+          owner = "tecnovert";
+          repo = "coincurve";
+          rev = "932366c9d4d8e487162b5c1b2a2d9693e24e0483";
+          hash = "sha256-zOekPmP1zR/S+zxq/7OrEz24k8SInlsB+wJ8kPlmqe4=";
+        };
+      });
+  bindir = linkFarm "bindir" (
+    lib.mapAttrs (_: p: "${lib.getBin p}/bin") {
+      particl = particl-core;
+      bitcoin = bitcoind;
+      namecoin = namecoind;
+      monero = monero-cli;
+      wownero = wownero;
```

Comment:
wownero currently fails to build because of a boost update.

Reply:
Rebased to include the fix.

Reply:
You don't necessarily need to rebase.